### PR TITLE
The original license needs to be added back to the project. Also corr…

### PR DESCRIPTION
**Pull request by davecramer**
…ect attribution for plcontainer.c to Lazlo

### CODE REVIEW

**Comment by paul-guo-** [comment]
@davecramer  Please do not remove the Pivotal licence header since lots of changes in the file are done by Pivotal employees. And is plpgj_core.c
https://github.com/codehaus/plj/blob/master/pl-j/csrc/plpgj_core.c? Assuming yes, do you have more context about why this information should be included given it seems that that file just includes the very common code which is used in other PL language code also. Thanks.

**Comment by davecramer** [comment]
@paul-guo- yes, that is the original file, and since you can't simply remove previous authorship it needs to be included. Unfortunately at the time I was a bit too aggressive when I removed the original authorship. Whether or not you can later copyright that file or not is up to the lawyers

**Comment by schubert** [comment]
@paul-guo- @yydzero I'll take this PR up. Let me take a look at it and I'll let you all know how to proceed.

**Comment by yydzero** [comment]
Thanks @schubert !

**Comment by davecramer** [comment]
Seems reasonable to me that both should be "Portions Copyright"

Dave Cramer

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Copyright 1994-2004 The PL-J Project. All rights reserved.
```
Comment (schubert) [inline LICENSE.txt]: Should be:

```
Portions Copyright 1994-2004 The PL-J Project. All rights reserved.
Portions Copyright © 2016-Present Pivotal Software, Inc.
```

**Comment by paul-guo-** [comment]
src/plcontainer.c should include Pivotal information also, right?

@schubert @davecramer

**Comment by davecramer** [comment]
@paul-guo- sure, makes sense.

**Comment by paul-guo-** [comment]
@davecramer  Please modify src/plcontainer.c and then it will be checked in.

**Pull request merged**